src/DotNet.RateLimiter/ActionFilters/RateLimitAttribute.cs

```diff
@@ -159,20 +160,14 @@ private string ProvideRateLimitKey(ActionExecutingContext context, string reques
             }
             if (!string.IsNullOrWhiteSpace(BodyParams))
             {
-                var parameters = BodyParams.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Why you removed this line? because by not having that, it won't work for multiple body params!